include/freerdp/codec/audio.h

```diff
@@ -69,6 +69,7 @@ typedef struct AUDIO_FORMAT AUDIO_FORMAT;
  * http://tools.ietf.org/html/rfc2361
  */
 
+#ifndef __MINGW32__
```

Comment:
Might be a good idea to prefix the `WAVE_FORMAT_*` symbols with `FREERDP_WAVE_FORMAT_*` if there is a collision

Reply:
I was wondering why not remove these. Changing symbols will effect whole project.

Reply:
because we need them, windows is not the only target ;)
changing at this point is no issue, as the next version will be `FreeRDP 3` as we already have broken too much to stay compatible with `2.0`